sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -13,21 +12,233 @@ namespace Azure.AI.TextAnalytics.Tests
+            string input = "Microsoft fue fundado por Bill Gates y Paul Allen.";
+
+            RecognizeEntitiesResult result = await client.RecognizeEntitiesAsync(input, "es");
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(3, entities.Count);
+        }
+
+        [Test]
+        public async Task RecognizeEntitiesWithSubtypeTest()
         {
-            string endpoint = Recording.GetVariableFromEnvironment("TEXT_ANALYTICS_ENDPOINT");
-            string subscriptionKey = Recording.GetVariableFromEnvironment("TEXT_ANALYTICS_SUBSCRIPTION_KEY");
+            TextAnalyticsClient client = GetClient();
+            string input = "I had a wonderful trip to Seattle last week.";
+
+            RecognizeEntitiesResult result = await client.RecognizeEntitiesAsync(input);
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
 
-            var options = Recording.InstrumentClientOptions(new TextAnalyticsClientOptions());
-            var client = new TextAnalyticsClient(new Uri(endpoint), new TextAnalyticsSubscriptionKeyCredential(subscriptionKey), options);
+            Assert.AreEqual(2, entities.Count);
 
-            return InstrumentClient(client);
+            foreach (NamedEntity entity in entities)
+            {
+                if (entity.Text == "last week")
+                    Assert.IsNotNull(entity.SubType);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "A developer with SSN 555-55-5555 whose phone number is 222-222-2222 is building tools with our APIs.";
+
+            RecognizePiiEntitiesResult result = await client.RecognizePiiEntitiesAsync(input);
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(2, entities.Count);
+
+            var entitiesList = new List<string> { "555-55-5555", " 222-222-2222 " };
+            foreach (NamedEntity entity in entities)
+            {
+                Assert.IsTrue(entitiesList.Contains(entity.Text));
+                Assert.IsNotNull(entity.Type);
+                Assert.IsNotNull(entity.Score);
+                Assert.IsNotNull(entity.Offset);
+                Assert.IsNotNull(entity.Length);
+                Assert.Greater(entity.Length, 0);
+            }
+        }
+
+        [Test]
+        public async Task RecognizePiiEntitiesWithLanguageTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string input = "A developer with SSN 555-55-5555 whose phone number is 222-222-2222 is building tools with our APIs.";
+
+            RecognizePiiEntitiesResult result = await client.RecognizePiiEntitiesAsync(input, "en");
+            IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
+
+            Assert.AreEqual(2, entities.Count);
```

Comment:
Do you care about content of these entities?

Reply:
This test is to make sure that when I set the language, I get results. 
`RecognizePiiEntitiesTest` will actually look that I was able to deserialize the response from the service